01-module-system/main.js

```diff
@@ -0,0 +1,65 @@
+const help = require('./helper')
+const a = help.getNotes()
+const chalk = require('chalk');
+const yargs = require('yargs')
+const addOption1= {
```

Comment:
leave empty lines after require statements. Also please rename addOptions1,2,3 to more convenient names (e.g. addOptions,  removeOptions, ...). Fix Whitespace across file